providers/flagd/README.md

```diff
@@ -69,6 +69,7 @@ Configuration can be provided as constructor options or as environment variables
 |----------------------------------------------------------|--------------------------------|-----------------------------|-----------|---------------------|
 | WithHost                                                 | FLAGD_HOST                     | string                      | localhost | rpc & in-process    |
 | WithPort                                                 | FLAGD_PORT                     | number                      | 8013      | rpc & in-process    |
+| WithTargetUri                                            | FLAGD_TARGET_URI               | string                      | ""        | in-process          |
```

Comment:
In Java (and the [spec](https://github.com/open-feature/flagd/pull/1432/files#diff-28ba4b4041b6c39c4b9deb542276312bebdd08f45b39b29d860849d9e8b81a6eR36) I've just opened) this env is different. I think we should make them consistent, and ideally stick with what's already been released in Java.

Reply:
I do think I like this name better though... we could just release a breaking changes to the Java provider @pradeepbbl @beeme1mr

Reply:
I agree with the point on consistency, should I align with `java` or as you suggest update the java provider. To be clear I don't think anyone using the custom resolver yet including in booking (will release once back from vacation).

Reply:
I think we should change it in Java.

Reply:
K I will do this quickly.

Reply:
https://github.com/open-feature/java-sdk-contrib/pull/1050

Reply:
thanks